specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/examples/NetworkVirtualApplianceListByResourceGroup.json

```diff
@@ -32,11 +32,16 @@
               "virtualHub": {
                 "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualHubs/hub1"
               },
+    		      "virtualApplianceSites": [
```

Comment:
>		 [](start = 4, length = 2)

nit - tabs and spaces mixed together. You have this problem in several examples.

Reply:
Done. Made changes to convert tabs to spaces in all the files